wagtail/admin/templates/wagtailadmin/pages/listing/_button_with_dropdown_new.html

```diff
@@ -0,0 +1,22 @@
+{% load wagtailadmin_tags i18n %}
+
+<div {{ self.attrs }} class="w-flex w-justify-center w-items-center {{ classes|join:' ' }}">
+    <button class="w-p-0 w-text-primary w-bg-transparent hover:w-scale-110 w-transition {{ button_classes|join:' ' }}" data-button-with-dropdown-toggle>
```

Comment:
I believe the prototype did have scaling on hover as well so we might have to check with the team on this one 🤔. (personally i don't mind it since its quite small)